docs/src/main/asciidoc/security-openid-connect-dev-services.adoc

```diff
@@ -126,6 +126,17 @@ Sometimes, ID tokens are forwarded to application frontends as bearer tokens.
 This helps endpoints identify the user logged into SPA or perform out-of-band token verification.
 Choose the *With ID Token* option in such cases.
 
+[NOTE]
+====
+When you import custom Keycloak realms, you may find, after inspecting both the access and ID tokens in Dev UI, that only the access token contains the list of roles in its `groups` claim.
+This information is important for accessing endpoints that are secured with the `@RolesAllowed` annotation.
+To tell Keycloak to include this information in the ID token as well, add the
+`microprofile-jwt` scope to the list of client scopes in the Keycloak admin console.
```

Comment:
```suggestion
To tell Keycloak to include this information in the ID token, add the `microprofile-jwt` scope to the list of client scopes in the Keycloak admin console.
```
Please avoid mid-sentence line breaks.
Removed "as well" because it is unclear what it refers to.